<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>561</width>
     <height>351</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>numEp</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>gold</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>silver</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>bronze</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="numEpIn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="numEp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>numEp</string>
   </property>
  </widget>
  <widget class="QLabel" name="goldLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gold</string>
   </property>
  </widget>
  <widget class="QLabel" name="silverLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Silver</string>
   </property>
  </widget>
  <widget class="QLabel" name="bronzeLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bronze</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="goldIn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="silverIn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="bronzeIn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton_Button">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add/Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton_Button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>addButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>removeButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addButton()</slot>
  <slot>removeButton()</slot>
 </slots>
</ui>
